RELEASE/scripts/autoscend/iotms/mr2023.ash

```diff
@@ -3,7 +3,7 @@
 boolean auto_haveRockGarden()
 {
 	static item rockGarden = $item[Packet Of Rock Seeds];
-	return auto_is_valid(rockGarden) && auto_get_campground() contains rockGarden;
+	return is_unrestricted(rockGarden) && auto_get_campground() contains rockGarden;
 }
```

Comment:
Same case as with other gardens, we don't need to use the seed packet item to pick the garden, and couldn't in G-Lover. We can however use groveling gravel, so should pick the garden if it's planted.